docs/src/get-started/tutorials/get-started.py

```diff
@@ -0,0 +1,172 @@
+
+Introduction to the rascaline interface using a molecular crystals dataset.
+
+The dataset
+-----------
+
+The dataset is a small subset dataset used in
+`ShiftML2 dataset <https://pubs.acs.org/doi/pdf/10.1021/acs.jpcc.2c03854>`_.
+containing molecular crystals dataset. There are four crystals-
+one with each of the elements [hydrogen, carbon],
+[hydrogen, carbon, nitrogen, oxygen], [hydrogen, carbon, nitrogen],
+and [hydrogen, carbon, oxygen].
+Each crystal has 10 structure, also denoted by frames, attributed to it.
+Adjacent numbers are adjacent frames. Frame 0 is the geometry-optimised frame.
+The following 9 frames contain atoms that are slightly displaced from
+the geometry-optimised frame. You can obtain the dataset from our
+:download:`website <../../../static/datasets.xyz>`.
+
+We first import the crucial packages and afterewards load the dataset
+using chemfiles.
+"""
+
+import chemfiles
+import numpy as np
+
+from rascaline import SphericalExpansion
+
+
+with chemfiles.Trajectory("../../../static/datasets.xyz") as trajectory:
+    frames = [f for f in trajectory]
+
+print(f"The dataset contains {len(frames)} frames.")
+
+# %%
+# We will not explain here how to use chemfiles in detail, as we only use a
+# few functions here. Briefly, :class:`chemfiles.Trajectory` loads structure
+# data in a convenient format into our example. If you want to learn more about
+# the possibilities take a look at the
+# `chemfiles documentation <https://chemfiles.org>`_.
+# Let us now take a look at the first frame of the dataset.
+
+frame0 = frames[0]
+
+print(frame0)
+
+# %%
+# With ``frame0.atoms``` we get a list of the atoms that make up the zero frame.
+# The `name` attribute returns to us the type of the specified atom.
+
+elements, counts = np.unique([atom.name for atom in frame0.atoms], return_counts=True)
+
+print(
+    f"The zero frame contains "
+    f"{counts[0]} {elements[0]}-atoms, "
+    f"{counts[1]} {elements[1]}-atoms, "
+    f"{counts[2]} {elements[2]}-atoms and "
+    f"{counts[3]} {elements[3]}-atoms."
+)
+
+# %%
+# Calculate a descriptor
+# ----------------------
+#
+# We will now calculate the atomic descriptor for this structure using the
+# SOAP spherical expansion as introduced by
+# `Bartók, Kondor and Csányi <http://dx.doi.org/10.1103/PhysRevB.87.184115>`_
+# For doing so we have to define a set parameters telling the rascaline how "detailed" the
+# spherical expansion should be calculated. These parameters are also called hyper parameters because
+# they are not being optimized in a usual machine learning procedure.
+# Hyper parameters are the crucial part of calculating descriptors. Poorly selected hyper parameters will
+# lead to a poor description of your dataset as discussed in
+# `literature <https://arxiv.org/abs/1502.02127>`_.
+#
+# Below are our default hyper parameters for this dataset.
+# The effect and of changing some hyper parameters are discussed in a different tutorial.
+
+HYPER_PARAMETERS = dict(
```

Comment:
I personally prefer using literal for dictionaries instead of the `dict` function, especially here where we have nested dictionaries

Reply:
I personally prefer the other, because it easily allows copying between parameters. But, I am fine with changing this.